presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -210,7 +212,9 @@ Limitations
 
 * Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
 * Push down of dynamic filters into local table scan on worker nodes is limited to broadcast joins.
-* Min-max dynamic filter collection is not supported for DOUBLE, REAL and unorderable data types.
+* Min-max dynamic filter collection is not supported for ``DOUBLE``, ``REAL`` and unorderable data types.
+* When ``IS NOT DISTINCT FROM`` is used in a join condition, dynamic filtering is not
+  supported if ``DOUBLE`` or ``REAL`` data types are used in the comparison expression.
```

Comment:
"Dynamic filtering is not supported for `DOUBLE` and `REAL` data types when using `IS NOT DISTINCT FROM` predicate."

?

Reply:
updated